src/main/java/io/appform/dropwizard/actors/actor/ActorConfig.java

```diff
@@ -96,6 +97,14 @@ public class ActorConfig {
     @Max(32)
     private Integer shardCount;
 
+    @Valid
+    @Builder.Default
+    private List<String> messageMetaGeneratorClasses = List.of(
+            "io.appform.dropwizard.actors.actor.metadata.MessageReDeliveredMetaGenerator",
+            "io.appform.dropwizard.actors.actor.metadata.MessageDelayMetaGenerator",
+            "io.appform.dropwizard.actors.actor.metadata.MessageExpiredMetaGenerator"
+    );
```

Comment:
Do not do this. Leave this empty. Create a const list of default metadata generators in this class and call it DEFAULT_METADATA_GENERATORS. Use that in the usage site, if this list is null only (not empty as this gives a way for people to disable this expensive operations if not needed)